modules/virt-adding-virtio-drivers-vm-yaml.adoc

```diff
@@ -43,18 +39,10 @@ volumes:
       image: container-native-virtualization/virtio-win
     name: virtiocontainerdisk
 ----
-<1> {VirtProductName} boots virtual machine disks in the order defined in the
-`VirtualMachine` configuration file. You can either define other disks for the
-virtual machine before the `container-native-virtualization/virtio-win` container disk or use the optional
-`bootOrder` parameter to ensure the virtual machine boots from the correct disk.
- If you specify the `bootOrder` for a disk, it must be specified for all disks
-in the configuration.
-
-. The disk is available once the virtual machine has started:
-** If you add the container disk to a running virtual machine, use
-`oc apply -f <vm.yaml>` in the CLI or reboot the virtual machine for the changes
-to take effect.
-** If the virtual machine is not running, use `virtctl start <vm>`.
-
-After the virtual machine has started, the VirtIO drivers can be installed from
-the attached SATA CD drive.
+<1> {VirtProductName} boots the VM disks in the order defined in the `VirtualMachine` manifest. You can define other VM disks that boot before the `container-native-virtualization/virtio-win` container disk or use the optional `bootOrder` parameter to ensure the VM boots from the correct disk. If you configure the boot order for a disk, you must configure the boot order for the other disks.
+
+. Apply the changes:
+* If the VM is running, run `oc apply -f <vm.yaml>` or reboot the VM.
```

Comment:
```suggestion
* If the VM is running, reboot the VM or run the following command:
+
[source, terminal]
----
$ run `oc apply -f <vm.yaml>`
----
```

Reply:
Not sure if you have a different style in the Virualization docs, but current OCP style is to list commands like this so they are easy for the customer to copy. For your consideration.

Reply:
Module was written 4 years ago, long before current OCP style. This is outside the scope of the change but I can try to clean it up.